chroma-manager/tests/integration/shared_storage_configuration/test_power_control.py

```diff
@@ -9,10 +9,16 @@ class ChromaPowerControlTestCase(ChromaIntegrationTestCase):
 
     def setUp(self):
         super(ChromaPowerControlTestCase, self).setUp()
+        
+        # Even though the tests only need 1 server, we need to add a server
```

Comment:
Isn't `self.TEST_SERVERS` already (i.e at lesat all four storage server in) `config['lustre_servers']` as defined `class ApiTestCaseWithTestReset`?

Reply:
Yes. But I'd rather not rely on superclass properties.